### PR TITLE
Update docker-cli-to-kubectl.md

### PR DESCRIPTION
**Pull request by msshroff**
In current page  "docker-cli-to-kubectl" the markdown file tags are jumbled up. 
Fixing the formatting and the sequence of commands, in section "add env to nginx-app"

### CODE REVIEW

**Comment by zparnold** [comment]
Hi there, I don't believe this was meant as a Markdown header but as a legit shell comment. Is there a reason you wanted to make it a header?

File: content/en/docs/reference/kubectl/docker-cli-to-kubectl.md [hunk 1]
```diff
@@ -37,22 +37,20 @@ kubectl:
...
-```shell
-# add env to nginx-app
-kubectl set env deployment/nginx-app  DOMAIN=cluster
 ```
 deployment.apps/nginx-app created
 ```
 
-```
+# add env to nginx-app
+kubectl set env deployment/nginx-app  DOMAIN=cluster
```
Comment (kbhawkey) [inline content/en/docs/reference/kubectl/docker-cli-to-kubectl.md:45]: Hello @msshroff .
Would you have look at lines 44-45? There appear to be duplicate commands listed.
Thanks!

Comment (msshroff) [reply content/en/docs/reference/kubectl/docker-cli-to-kubectl.md:45]: please ignore this PR. 
The issues are fixed by commit 82546d3ff88742d0209f78676f17a69f5ff03545 already.

**Comment by msshroff** [comment]
please ignore this PR.
The issues are fixed by commit 82546d3 already.

Duplicate of #82546d3

**Comment by kbhawkey** [comment]
:wave @msshroff .
Would you like to close this pull request?

**Comment by msshroff** [comment]
closing as duplicate

**Pull request closed without merging**